instrumentation/http-url-connection/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/httpurlconnection/HttpUrlNetAttributesGetter.java

```diff
@@ -18,6 +18,20 @@ public String getTransport(HttpURLConnection connection, @Nullable Integer statu
     return SemanticAttributes.NetTransportValues.IP_TCP;
   }
 
+  @Nullable
+  @Override
+  public String getProtocolName(HttpURLConnection connection, @Nullable Integer integer) {
+    // HttpURLConnection hardcodes the protocol name&version
+    return "http";
```

Comment:
should we not capture since this is the default? cc @lmolkova

Reply:
yep, great catch!

Reply:
Just to clarify: should we not capture `http` at all; not capture `http` `1.1`; or something else?

Reply:
my reading is that we should capture the version (`1.1`) if we know it, but not the protocol

Reply:
Hm, why? The `net.protocol.name` attribute is (or will be) shared across different semantic conventions, it's not like it has a constant default value that we can just skip (like `net.sock.family` has `inet`). IMO knowing that version is `1.1` without knowing the name of protocol sends a somewhat unclear message - you can guess that it actually is `http` by looking at the other attributes though.

Reply:
Interesting. So we don't have a good way to enforce defaults for net attributes on higher-level instrumentations, right? 
I.e. populating `net.protocol.version = http` on HTTP spans is pure redundancy, but then we'd have to have http-specific implementation of net attribute extractor

Reply:
> I.e. populating `net.protocol.version = http` on HTTP spans is pure redundancy, but then we'd have to have http-specific implementation of net attribute extractor

I'm actually thinking about having the HTTP getter extend the net getter (just as http semconv kind of "extends" the net semconv); but even in that case I wouldn't say that it's 100% redundant. There are some HTTP clients that support QUIC or SPDY - and while I think that almost nobody actually uses these protocols, there is still a possibility to use them, and the instrumentation should be prepared to reflect that.